test.js

```diff
@@ -216,7 +216,7 @@ test(
       },
     });
     t.true(
-      stderr.includes(
+      stdout.includes(
```

Comment:
something weird is going on here that, despite being thrown as `throw new serverless.classes.Error` it is being output on stdout instead of stderr.